chromiumcontent/BUILD.gn

```diff
@@ -357,57 +357,57 @@ if (is_electron_build && !is_component_build) {
 
     deps = [
       "//base:base_unittests",
-      "//cc:cc_unittests",
-      "//cc/blink:cc_blink_unittests",
-      "//content/test:content_browsertests",
-      "//content/test:content_unittests",
-      "//crypto:crypto_unittests",
-      "//device:device_unittests",
-      "//gin:gin_unittests",
-      "//gpu:gpu_unittests",
-      "//gpu/ipc/service:gpu_ipc_service_unittests",
-      "//ipc:ipc_tests",
-      "//media:media_unittests",
-      "//media/midi:midi_unittests",
-      "//media/mojo:media_mojo_unittests",
-      "//media/capture:capture_unittests",
-      "//mojo/common:mojo_common_unittests",
-      "//mojo/edk/system:mojo_system_unittests",
-      "//mojo/edk/test:mojo_public_bindings_unittests",
-      "//mojo/edk/test:mojo_public_system_unittests",
-      "//net:net_unittests",
-      "//ppapi:ppapi_unittests",
-      "//printing:printing_unittests",
-      "//skia:skia_unittests",
-      "//storage:storage_unittests",
-      "//sql:sql_unittests",
-      "//third_party/angle/src/tests:angle_unittests",
-      "//third_party/leveldatabase:env_chromium_unittests",
-      "//third_party/WebKit/public:test_support",
-      #"//third_party/WebKit/public:all_blink", #TODO: Fix build
-      "//tools/gn:gn_unittests",
-      "//ui/base:ui_base_unittests",
-      #"//ui/compositor:compositor_unittests", # these are failing in chromium
-      "//ui/display:display_unittests",
-      "//ui/events:events_unittests",
-      "//ui/gl:gl_unittests",
-      "//url:url_unittests",
-      "//url/ipc:url_ipc_unittests",
-      "//v8/test/unittests:unittests",
+      # "//cc:cc_unittests",
```

Comment:
What's the plan for the rest of these tests; will these be uncommented at some point?

Reply:
@ckerr this PR changes the tests to only build/run the `base:base_unittests` tests because of timeout issues building/running everything (it was timing out after running for **5** hours!).  I figure we can (in another PR) adjust the tests to pick the ones we actually want to run.